scripts/west_commands/zspdx/sbom.py

```diff
@@ -121,4 +121,10 @@ def makeSPDX(cfg):
         log.err("SPDX writer failed for build document; bailing")
         return False
 
+    # write modules document
+    writeSPDX(os.path.join(cfg.spdxDir, "modules-deps.spdx"), w.docModulesExtRefs)
+    if not retval:
+        log.err("SPDX writer failed for build document; bailing")
```

Comment:
Minor nit, I'd probably change "build document" to "modules dependencies document" or something like that, for the error message here.